dev_guide/pod_autoscaling.adoc

```diff
@@ -0,0 +1,111 @@
+= Pod Autoscaling
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+A *HorizontalPodAutoscaler* object specifies how the system should automatically
+increase or decrease the scale of a *ReplicationController* or *DeploymentConfig*,
+based on metrics collected from the pods that belong to that *ReplicationController*
+or *DeploymentConfig*.
+
+== Requirements for Using Horizontal Pod Autoscalers
+
+In order to use *HorizontalPodAutoscalers*, your cluster admin must have
+link:../admin_guide/cluster_metrics.html[set up Heapster for cluster metrics],
+running in the `openshift-infra` namespace.
+
+== Metrics Supported
+
+The following describes the metrics supported by *HorizontalPodAutoscalers*:
+
+.Metrics
+[cols="3a,8a",options="header"]
+|===
+
+|Metric |Description
+
+|CPU Utilization
+|Percentage of the link:../dev_guide/compute_resources.html#cpu-requests[requested CPU]
+|===
```

Comment:
@aveshagarwal If you're viewing in the GitHub preview only, you'll need to change the path to .adoc. In the actual docs build, .html is correct.